content/applications/sales/crm/track_leads/lead_scoring.rst

```diff
@@ -30,6 +30,20 @@ Specifically, Odoo's predictive lead scoring uses the *naive Bayes* probability
    P(A | B) = \frac{P(A) \times P(B | A)}{P(B)}
    \end{equation}
 
+Breaking down the equation:
+
+- P(A|B) = The probability of a successful lead in this *case*
+- P(A) = The overall probability of a lead being successful regardless of the conditions
+- P(B|A) = The probability of this being the *case* given a lead is successful
+- P(B) = The probability of this being the *case*
```

Comment:
```suggestion
- P(A|B) = probability of a successful lead in this *case*
- P(A) = overall probability of a lead being successful, regardless of the conditions
- P(B|A) = probability of this being the *case*, given a lead is successful
- P(B) = probability of this being the *case*
```
removing the capitalized "The" since there is no period at the end of these explanations, nor are they complete sentences.